GameData/RealismOverhaul/Parts/Avionics/Sounding0-3m.cfg

```diff
@@ -139,34 +118,28 @@
 		name = kOSProcessor
 		// Low space high power core for early sounding rockets.
 		diskSpace = 100
-		diskSpaceCostFactor = 1         // 100F for double space
-		diskSpaceMassFactor = 0.001     // 100 kg for double space
-		ECPerInstruction = 0.000001     // 1kW @ 2000 IPU
-    }
+		diskSpaceCostFactor = 1			// 100F for double space
+		diskSpaceMassFactor = 0.001		// 100 kg for double space
+		ECPerInstruction = 0.000001		// 1kW @ 2000 IPU
+	}
 }
 
 
-//  Aerobee Sounding Rocket
-//  Dimensions: 0.38 m x 0.12 m
-//  Inert Mass: 60 Kg
+//	Aerobee Sounding Rocket
+//	Dimensions: 0.38 m x 0.26 m
+//	Inert Mass: 60 Kg
 +PART[RP0probeSounding0-3m]:AFTER[RealismOverhaul]
 {
 	@name = RP0aerobeeSounding
 
 	@MODEL
 	{
-		%scale = 0.15, 0.252632, 0.15
+		%scale = 0.3086, 1.0, 0.3086
 	}
 
-	%node_stack_top = 0, 0.063158, 0, 0.0, 1.0, 0.0, 0
-	%node_stack_bottom = 0, -0.063158, 0, 0.0, -1.0, 0.0, 0
+	%rescaleFactor = 0.48336
 
-	@DRAG_CUBE
-	{
-		@rescaleX = 0.15
-		@rescaleY = 0.252632
-		@rescaleZ = 0.15
-	}
+	%rescaleCube = 0.48336
```

Comment:
`rescaleCube` doesn't do what you hoped.  It is just a flag for if the cube should get rescaled by the `rescaleX`/`rescaleY`/`rescaleZ` values inside the DRAG_CUBE node, it does not treat as if those individual values = the `rescaleCube` value.

Please either just delete the DRAG_CUBE node and allow the PartLoader to auto-generate one, OR to preserve the hollow-ness of the part, edit such that 
```
rescaleX = #$../MODEL/scale[0]$
rescaleY = #$../MODEL/scale[1]$
rescaleZ = #$../MODEL/scale[2]$
@rescaleX *= #$../rescaleFactor$
@rescaleY *= #$../rescaleFactor$
@rescaleZ *= #$../rescaleFactor$
```
I _think_ that will do the correct thing.